deploy/deployment.yaml

```diff
@@ -3,10 +3,20 @@ apiVersion: v1
 metadata:
```

Comment:
the file does not only contains a "templated deployment", so maybe we should rename it?

Reply:
do you mean to rename the file? yes, I can rename it - see my proposal here: https://github.com/codeready-toolchain/api/pull/65#discussion_r352639132

Reply:
renamed 8025b63